templates/AP_app/navbar.html

```diff
@@ -13,7 +13,7 @@
                     {% trans "Management" %}
                 </a>
                 <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-                    <a class="dropdown-item" href="{% url 'invoices-list' %}"><i
+                    <a class="dropdown-item" href="{% url 'invoices-list' %}?status=2"><i
```

Comment:
It's better if you don't use hardcoded values, try to bind values with get_context_data